src/main/webapp/profLoader.js

```diff
@@ -11,14 +11,30 @@
   const profKey = urlParams.get("prof-key");
   const profName = urlParams.get("prof-name");
 
+  function fillTitle(profData) {
+    document.getElementById("prof-name").innerHTML = profName;
+    linkContainer = document.getElementById("link-container");
```

Comment:
Is this a local variable, or a global you declare somewhere? If the latter I'm not seeing it.
I'd prefer to declare this here as a const and just re-fetch from the document if you need it elsewhere.

FYI JavaScript lets you declare variables with no var/etc in front, but they become vars by default.